### PR TITLE
Flow Build assets from AppAuthentication

### PR DESCRIPTION
**Pull request by pakrym**
https://github.com/aspnet/AzureIntegration/issues/186

### CODE REVIEW

File: src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/Microsoft.AspNetCore.AzureKeyVault.HostingStartup.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(MicrosoftAspNetCoreHostingAbstractionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="$(MicrosoftExtensionsConfigurationEnvironmentVariablesPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="$(MicrosoftExtensionsConfigurationAzureKeyVaultPackageVersion)" />
-    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="$(MicrosoftAzureServicesAppAuthenticationPackageVersion)" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="$(MicrosoftAzureServicesAppAuthenticationPackageVersion)" PrivateAssets="Analyzers" />
```
Comment (natemcmaster) [inline src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/Microsoft.AspNetCore.AzureKeyVault.HostingStartup.csproj]: Why not just PrivateAssets='None'?

**Comment by natemcmaster** [comment]
cc @muratg do we need patch approval for this? The package hasn't RTM-ed yet

File: src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/Microsoft.AspNetCore.AzureKeyVault.HostingStartup.csproj (hunk 1 above)
Comment (pakrym) [reply src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/Microsoft.AspNetCore.AzureKeyVault.HostingStartup.csproj]: The ask was no to include analyzers.

**Comment by natemcmaster** [comment]
```xml
<PackageArtifact Include="Microsoft.AspNetCore.AzureKeyVault.HostingStartup" Category="ship" />
```
Yes, it was marked ship. https://github.com/aspnet/Universe/blob/release/2.0/build/artifacts.props#L42

I can manually exclude from ship output for this build

**Comment by muratg** [comment]
@natemcmaster OK, sounds good. Thanks.

**Comment by pakrym** [comment]
It would need to be ship after initial oob release.

**Comment by pakrym** [comment]
:up::date:

**Pull request merged**